plugin.video.mediathek/addon.xml

```diff
@@ -37,6 +37,7 @@ Al momento sono supportate le seguenti emittenti:
     <website>https://github.com/raptor2101/Mediathek</website>
     <source>https://github.com/raptor2101/Mediathek</source>
     <email>raptor2101@gmx.de</email>
+    <broken>For updates of Mediathek, please upgrade to leia or above</broken>
```

Comment:
Could you change the description why your add-on is broken?

Reply:
I agree here. As we are closing https://github.com/xbmc/repo-plugins/pull/2736 this message is no longer valid.

Reply:
As stated by @sarbes "Sorry, but in its current state, the add-on has no chance of being merged.". All points stated for #2736 goes for this version of the plugin as well and it won't get futher updates from my side.

Reply:
I understand that, but the current message `For updates of Mediathek, please upgrade to leia or above` is not correct now. It would be better to add a proper reason? Otherwise users will start looking for the new version.

Reply:
Than just adjust the message... any further discussion is more time consuming than this edit. I'am currently at work and so i'am unable to make a new PR. Close this one and remove the Plugin from the repository.